### DISCUSSION TITLE
Is there any way to validate the uploaded file during uploading(runtime) #21

### DISCUSSION THREAD

**Discussion by melhalees** [spatie/laravel-medialibrary#2868] [Q&A] [answered]
Hello,

I am wondering if this package could validate the uploaded file before uploading the file because someone can upload a wrong mime type with a big size and after waiting some time, he will figure that his uploaded file does not meet the requirement that I have defined in the backend.

Thanks.

**Comment by PaolaRuby** [comment, marked as answer]
Set valid extensions on input file ` accept=".xls,.xlsx"`, or validate with js before submit

**Reply by melhalees** [reply]
Thanks, I did it by writing some javascript code.
